.gitignore

```
@@ -17,4 +17,8 @@ meshery-consul
 
 log.txt
 
-.idea
\ No newline at end of file
+.idea
+
+errorutil_analyze_errors.json
+errorutil_analyze_summary.json
+errorutil_errors_export.json
```

```suggestion
**errorutil_analyze_errors.json
**errorutil_analyze_summary.json
**errorutil_errors_export.json
```
Let's modify these a little so that these files are ignored in all dirs and sbudirs they're in.

---

On it